packages/transformers/html/src/dependencies.js

```diff
@@ -160,6 +160,11 @@ export default function collectDependencies(asset: MutableAsset, ast: AST) {
         continue;
       }
 
+      // Check for id references
+      if (attrs[attr][0] === '#') {
```

Comment:
I think this should also check `tag === 'use' && (attr === 'href' || attr === 'xlink:href')`?

Reply:
Yea, I thought about that, but isn't always hash at the beginning of the `src`/`href`/any other "path-related" attribute means an id-based selector?

Reply:
For `<a href="#xyz">` yes, but `<img src="#xyz">` should probably still be an error.

Reply:
Yeees and no. `<img src="#">` isn't invalid (on the other hand, `<img src="">` is treated as invalid by the browsers) and because of that it's even sometimes used as a "placeholder src value" when e.g. the `src` is meant to be later changed by js (https://stackoverflow.com/questions/19613841/img-src-means-in-html).

It's not a strong use-case, that's true, but I'm not sure if parcel should fail on a valid HTML? But also, you're the parcel member here, so if you say that I should check for `use` element & attr name here, that's what I will do

Reply:
Out of curiosity [I've hacked a small codepen](codepen.io/kkkubas/pen/zYqXvVO) and tried out all of the elements & attributes which are being handled by this html transformer with id-based path: `#sth`.
It seems that none of the elements is failing (at least not because of having the id-based path set - I've commented out `script` tag under `ATTRS.src`, because the `script` element have tried to load the current page as an js script - which obviously have failed).

BTW. surprisingly the `object` element have loaded the empty HTML page when given `#sth` as the `data` - the same way as `iframe` did. I didn't know it would work like this 😄

Reply:
Hey @mischnic what would be the decision here? Let's limit the condition to check for tag & attr name and call it a day?
Or do we leave current solution?